openwell/questioner
# #163445913 implementation logic feedback

#### What does this PR do?
- This PR integrates improved logic for better performance 
#### Description of Task to be completed?
-  Accessing of API end points
#### How should this be manually tested?
- Pull this branch [ch-implementation-logic-feedback-163445913](https://github.com/openwell/questioner/tree/ch-implementation-logic-feedback-163445913) off this repository
- On your command line, run  `npm install` to install all dependencies
- Run `npm database` to create the database tables and create an admin
- An admin token is generated if successfully and will be displayed in the terminal
- The token should be part of the headers as 'tokens': 'token-code'
- Run `npm start` to start the app
- The app would run on port 3000
- Access the endpoint on `localhost:3000/api/v1/`
#### Any background context you want to provide?
- This app is a node app and would require nodejs and npm installation on your local machine
- This app will require PostgreSQL Database
#### What are the relevant pivotal tracker stories?
[#163445913](https://www.pivotaltracker.com/story/show/163445913)
#### Screenshots (if appropriate)
None
#### Questions:
None
